README.md

```diff
@@ -143,105 +143,16 @@ The following commands are supported for profiles:
 | `SAS.updateProfile` | SAS: Update Connection profile         |
```

Comment:
looks like the connect-and-run.md also includes this table, so just wondering do we still want this table here?

Reply:
I think I'd like to keep it in both places for reference.